core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -394,12 +397,14 @@ export class LoggedOutUser extends BaseUser {
   async clickAboutButtonInAboutMenuOnNavbar(): Promise<void> {
     if (this.isViewportAtMobileWidth()) {
       await this.clickOn(mobileNavbarOpenSidebarButton);
+      await this.clickOn(mobileSidebarExpandAboutMenuButton);
       await this.clickButtonToNavigateToNewPage(
         mobileSidebarAboutButton,
         'About Oppia button in the About Menu on mobile sidebar',
         aboutUrl,
         'About'
       );
+      showMessage('Clicked on the About Oppia button in the mobile sidebar.');
```

Comment:
For the consistency of all functions, I think it's better to take this showMessage out since some functions don't have this success message.